lib/scion.js

```diff
@@ -1178,3 +1178,5 @@
         InterpreterScriptingContext : InterpreterScriptingContext
     };
 }));
+
+var scion=this.SCION;
```

Comment:
UMD format already creates a global variable called 'scion'. This line: https://github.com/jbeard4/SCION-CORE/blob/master/lib/scion.js#L27